packages/sdk/src/identity/identity.interface.ts

```diff
@@ -0,0 +1,30 @@
+import type { PublicKey } from '@solana/web3.js';
+
+export interface Identity {
+  identityName: string;
```

Comment:
May be type or source would be better naming? up to you

Reply:
agree, changed to `type`